olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -552,5 +553,5 @@ spec:
       image: registry.svc.ci.openshift.org/openshift/knative-v0.14.2:knative-eventing-channel-dispatcher
     - name: IMAGE_KN_CLI_ARTIFACTS
       image: registry.svc.ci.openshift.org/openshift/knative-v0.14.0:kn-cli-artifacts
-  replaces: serverless-operator.v1.7.1
-  version: "1.8.0-rc1"
+  replaces: serverless-operator.v1.7.2
```

Comment:
This is needed, otherwise the catalog is broken.

Reply:
Oh, so when we have skipRange we also need `replaces:` ? Then I'm curious what will happen when we have 1.7.3 which also `replaces` 1.7.2. I think that's not allowed by OLM.

Reply:
I've seen some CSV's with `skipRange` _and_ `replaces`, and some with just `skipRange`, and obviously, ones with just `replaces`. I don't know which one is right. I'll dig in.

Reply:
According to Shawn, replaces is not required if skipRange is specified.

Reply:
Just too loop back on this, based on Slack threads, it seems `replaces` IS required.

Reply:
That thread was updated to say just having skipRange is "10000%" a valid thing. `replaces` is not required.

Reply:
@warrenvw that is not true. That is only true (as said in the thread) if the respective CSV is the **only** one in a channel!

Reply:
👍